packages/react/src/auto/mui/MUIAutoForm.tsx

```diff
@@ -36,7 +38,7 @@ export const MUIAutoForm = <
 >(
   props: MUIAutoFormProps<GivenOptions, SchemaT, ActionFunc, Options>
 ) => {
-  const { action, include: _include, exclude: _exclude, fields: _fields, submitLabel: _submitLabel, ...rest } = props;
+  const { action, include: _include, exclude: _exclude, fields: _fields, submitLabel: _submitLabel, record, findBy, ...rest } = props;
```

Comment:
Unless I'm missing something, I don't think we need to destructure these include/exclude props here.

Reply:
Originally I made this match with the Polaris equivalent component. I removed the unused de-structured properties in both cases.

It seems like this was here before `useFormFields` handled field inclusion/exclusion and it was not removed. Good catch